api/routes/wards.rb

```diff
@@ -3,7 +3,7 @@
 class App < Roda
   WHITELIST_PARAMS = %w[human_readable_name contactdata email password].freeze
 
-  route('wards') do |r|
+  route('wards', 'auth_api') do |r|
     # route[List_wards]: /api/wards
```

Comment:
Am I mistaken or should that change to
```ruby
# route[List_wards]: /api/wards/auth_api 
```

Reply:
You are "mistaken". auth_api in not part of the route. Is just a **namespace**. In app.rb, there is a route("api") do r.multiroute("auth_api") that will route here, but the URL is still /api/wards.

Reply:
The real URLs are shown at https://github.com/TheRealWolkenbruch/are-you-there-api/pull/19/files#diff-7919cb394053df1b5b8de7bfbd4d0a90R14